File: api/PersonsAPI/Entities/Person.cs
namespace PersonsAPI.Entities
{
    public class Person
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string Phone { get; set; }
        public string Email { get; set; }
        public string Address { get; set; }
        public string PostalZip { get; set; }
        public string Country { get; set; }
    }
}

File: app/src/models/person.ts
export interface Person {
  id: string;
  name: string;
  phone: string;
  email: string;
  address: string;
  postalZip: string;
  country: string;
}

export interface ApiResponseCollection {
  data: Person[];
  isSuccess: boolean;
  statusCode: number;
  message: string;
  errors?: any;
}

export interface ApiResponse {
  data: Person;
  isSuccess: boolean;
  statusCode: number;
  message: string;
  errors?: any;
}
